README.md

```diff
@@ -113,12 +111,14 @@ Within the container, build the font:
     $ make -C src/icons
 
 <!--- reference links for badges -->
-[all-contrib]: https://img.shields.io/badge/all_contributors-125-orange.svg?style=flat-square "All Contributors badge"
+[all-contrib]: https://img.shields.io/badge/all_contributors-125-orange.svg "All Contributors badge"
 [build-status-badge]: https://travis-ci.org/ForkAwesome/Fork-Awesome.svg?branch=master "Build status badge"
 [build-status-link]: https://travis-ci.org/ForkAwesome/Fork-Awesome
-[cdnjs-badge]: https://img.shields.io/cdnjs/v/fork-awesome.svg?style=flat-square "CDNJS badge"
+[cdnjs-badge]: https://img.shields.io/cdnjs/v/fork-awesome.svg "CDNJS badge"
 [cdnjs-link]: https://cdnjs.com/libraries/fork-awesome
-[jsdeliver-badge]: https://data.jsdelivr.com/v1/package/npm/fork-awesome/badge "JSDeliver badge"
+[jsdeliver-badge]: https://data.jsdelivr.com/v1/package/npm/fork-awesome/badge?style=rounded "JSDeliver badge"
 [jsdeliver-link]: https://www.jsdelivr.com/package/npm/fork-awesome
-[npm-badge]: https://img.shields.io/npm/v/fork-awesome.svg?style=flat&colorB=CB3837 "NPM badge"
-[npm-link]: ttps://www.npmjs.com/package/fork-awesome
+[npm-badge]: https://img.shields.io/npm/v/fork-awesome.svg?colorB=CB3837 "NPM badge"
+[npm-link]: https://www.npmjs.com/package/fork-awesome
+[chat-badge]: https://img.shields.io/matrix/forkawesome:matrix.org?label=chat%20%28matrix%29 "chat (matrix) badge"
```

Comment:
```suggestion
[chat-badge]: https://img.shields.io/matrix/forkawesome:matrix.org?label=%23forkawesome%3Amatrix.org "chat (matrix) badge"
```

makes the badge accessible.

Reply:
Maybe you're right, that everyone knows about matrix by now? I just thought my original formulation would be clearer for anyone who hadn't heard of matrix by now.

Reply:
_matrix_ is still retained. and the `#<channel-name>` is mainstream now ( thanks to Slack ). people ( even the ones who has not heard of matrix before ) would at least read "_channel forkawesome on service called matrix.org_"